sql/moz-fx-data-shared-prod/telemetry_derived/firefox_use_counters_v1/query.sql

```diff
@@ -0,0 +1,16623 @@
+-- Query for telemetry_derived.firefox_use_counters_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+---------SECTION 1 - FIREFOX DESKTOP ---------
+WITH firefox_desktop_use_counts_by_day_version_and_country_stg AS (
+  SELECT
+    DATE(a.submission_timestamp) AS received_date,
+    mozfun.norm.truncate_version(client_info.app_display_version, 'major') AS version_major,
+    metadata.geo.country AS geo_country,
+    normalized_app_name AS platform,
```

Comment:
Is `platform` field something we refer to `normalized_app_name` app in the context of this data?

Reply:
That is a good question - I'm open to either option, the data requestor asked for it to be called "platform" initially which is why I went with that to start.  (Maybe because this dataset is going to be a public facing one, so maybe platform is more easy to understand, i.e. Fenix or Desktop)